cmd/hegel/Dockerfile

```diff
@@ -1,7 +1,17 @@
-FROM golang:1.17.5-alpine
-COPY . /usr/myapp
-WORKDIR /usr/myapp
-RUN mkdir -p bin && go build -o ./bin ./...
+FROM golang:1.17.5-alpine AS builder
```

Comment:
go.mod is set to 1.13, that should get a bump, maybe a separate small PR though

Reply:
Agree